app/lib/meadow/indexing/v2/file_set.ex

```diff
@@ -11,6 +11,7 @@ defmodule Meadow.Indexing.V2.FileSet do
       accession_number: file_set.accession_number,
       api_link: Path.join([api_url(), "file-sets", file_set.id]),
       api_model: "FileSet",
+      canonical_link: Path.join([dc_url(), "items", file_set.work_id]),
```

Comment:
My inclination would be to leave this off the file set?

Reply:
Agreed, unless there's a reason we need it (I can't think of one at the moment). We could always add it in later too.

Reply:
That’s fine. I figured it was good to be consistent across types, but I’m not fussy about it. I’ll take it out.